modules/detectron/ps_roi_pool_op.cu

```diff
@@ -123,8 +123,13 @@ __global__ void PSRoIPoolForward(
       roundf(offset_bottom_rois[4]) + 1.) * spatial_scale;
 
     // Force too small ROIs to be 1x1
+    #ifdef __HIP_PLATFORM_HCC__
+    T roi_width = fmax(roi_end_w - roi_start_w, 0.1);  // avoid 0
```

Comment:
shouldn't this be `fmaxf` as the data types are float?

Reply:
Hi @rohithkrn , I guess we can use either fmaxf or fmax, but the reason I prefer fmax because 0.1 is with no suffix which defines double (https://en.cppreference.com/w/cpp/language/floating_literal).
Earlier it shows the following warning message:
/root/pytorch/modules/detectron/hip/ps_roi_pool_op_hip.cc:127:48: warning: implicit conversion from 'double' to 'int' changes value from 0.1 to 0 [-Wliteral-conversion]
    T roi_width = max(roi_end_w - roi_start_w, 0.1);  // avoid 0
                  ~~~                          ^~~
please let me know if we prefer fmaxf, I'll update it